smart-contracts/test/Lock/shareKey.js

```diff
@@ -216,7 +216,8 @@ contract('Lock / shareKey', accounts => {
     })
 
     it('should not assign the recipient of the granted key as the owner of tokenId 0', async () => {
-      await reverts(lock.ownerOf.call(0), 'NO_SUCH_KEY')
+      const zeroOwner = await lock.ownerOf.call(0)
+      assert.notEqual(zeroOwner, accountWithNoKey2)
```

Comment:
instead of notEqual maybe we should test to say it does equal 0x0

Reply:
done.